pkgs/by-name/cr/cryfs/package.nix

```
@@ -64,13 +63,11 @@ stdenv.mkDerivation rec {
     spdlog
   ] ++ lib.optional stdenv.cc.isClang llvmPackages.openmp;
 
```

```suggestion
  buildInputs = [ gtest ];
```
Did you manage to build this on Linux? CMake errors out with a missing `gtest` error.